book/working_with_strings.md

```diff
@@ -72,7 +57,24 @@ Nushell currently supports the following escape characters:
 - `\t` - tab
 - `\u{X...}` - a single unicode character, where X... is 1-6 hex digits (0-9, A-F)
 
-## Bare strings
+## Raw strings
+
+Raw strings behave the same as a single quoted strings, except that raw strings may also contain single quotes.
+This is possible because raw strings are enclosed by a starting `r#'` and a closing `'#`. This syntax should look familiar to users of Rust.
+
+```nu
+r#'some text'# == 'some text' # true
```

Comment:
This is a bit confusing with the == blah blah. I wonder if you can do something like
```nushell
r#'some text'#
```
would output
```
some text
```

Reply:
Yeah, I was lazy - I just copied the release notes that introduced the feature ;-).  I've updated the examples.